crates/bevy_reflect/src/type_registry.rs

```diff
@@ -1,5 +1,5 @@
 use crate::Reflect;
```

Comment:
There's some tests in this file related to get_short_name. Should those be deleted/moved?

Reply:
Yeah, I think we can safely yeet them now.